cibyl/sources/elasticsearch/api.py

```diff
@@ -355,6 +364,125 @@ def get_deployment(self, **kwargs):
 
         return AttributeDictValue("jobs", attr_type=Job, value=job_objects)
 
+    @speed_index({'base': 3})
+    def get_tests(self, **kwargs):
+        """
+            Get tests for a elasticsearch job.
+
+            :returns: container of jobs with the last completed build
+            (if any) and the tests
+            :rtype: :class:`AttributeDictValue`
+        """
+        jobs_found = self.get_jobs(**kwargs)
+        query_body = {
+            "query": {
+                "bool": {
+                    "must": [
+                        {},
+                        {
+                            "bool": {
+                              "should": [
+                                {}
+                              ]
+                            }
+                        },
+                        {
+                            "exists": {
+                                "field": "test_status"
+                            }
+                        },
+                        {
+                            "exists": {
+                                "field": "test_name"
+                            }
+                        }
+                    ]
+                }
+            },
+            "sort": [{"timestamp.keyword": {"order": "desc"}}]
+        }
+
+        build_numbers = []
+        if 'build_number' in kwargs:
+            build_numbers = [status.upper()
+                             for status in
+                             kwargs.get('build_number').value]
```

Comment:
If 'build_number' is not on kwargs, this will crash because you will try to access an attribute on a None object.

Reply:
Can you clarify this case and the command?
```
cibyl -vvv -d --sources elasticsearch --jobs **name_test** --tests
```
It doesn't crash and it's implemented in other previous methods.

Reply:
Sorry, my bad. You already checked that the argument is there on line 406, so what I said is not going to happen.